src/rastervision/tagging/tasks/validation_eval.py

```diff
@@ -18,13 +20,21 @@
 
 class Scores():
     """A set of scores for the performance of a model on a dataset."""
-    def __init__(self, y_true, y_pred, all_tags):
+    def __init__(self, y_true, y_pred, dataset):
         self.f2 = fbeta_score(y_true, y_pred, beta=2, average='samples')
+        self.f2_weighted = fbeta_score(y_true, y_pred, beta=2,
+                                       average='weighted')
```

Comment:
I think we want to use the `macro` option for `average`. This will not account for unbalanced labels which is what we want.